src/routes/api/trips.route.js

```diff
@@ -3,12 +3,18 @@ import { Trips } from '../../controllers';
 import { Authentication } from '../../middlewares';
 import { multicityCheck, validateInput } from '../../validation';
 
-const { multiCityRequest } = Trips;
 const { authenticate } = Authentication;
 
 
 const router = express.Router();
 
-router.post('/request', authenticate, multicityCheck, validateInput, multiCityRequest);
+const tripRoute = (io = null) => {
+  const notify = new Trips(io);
```

Comment:
the variable name here doesn't correlate with the class. The class of `Trips` is now `notify`?